utils/clflags.ml

```diff
@@ -142,13 +143,40 @@ let flambda_invariant_checks =
 
 let dont_write_files = ref false        (* set to true under ocamldoc *)
 
-let std_include_flag prefix =
-  if !no_std_include then ""
-  else (prefix ^ (Filename.quote Config.standard_library))
+let libs_in_path () =
+  if !no_stdlib
+  then []
+  else
+    "stdlib" ::
+      (if !no_stdinclude
+       then []
+       else [
+         "bigarray";
+         "dynlink";
+         "graphics";
+         "raw_spacetime_lib";
+         "str";
+         "unix";
+       ])
 ;;
 
-let std_include_dir () =
-  if !no_std_include then [] else [Config.standard_library]
+let std_include_flags prefix =
+  List.map (fun lib ->
+    let dir =
+      match lib with
+      | "stdlib" -> Config.standard_library
+      | _ -> Filename.concat Config.standard_library lib
+    in
+    prefix ^ Filename.quote dir
+  ) (libs_in_path ())
+;;
```

Comment:
I would write
```ocaml
let std_include_flags prefix =
  List.map (fun dir -> prefix ^ Filename.quote dir) (std_include_dirs ())
```
for less repetition.

Reply:
Indeed, not sure why I didn't write it this way in the first place.